cvat/apps/annotation/yolo.py

```diff
@@ -24,98 +24,46 @@
 
 def load(file_object, annotations):
```

Comment:
@zhiltsov-max , I don't think that the file should exist at all. The behaviour should be similar to 'dataset export' functionality. But let's go step by step.